src/shared/components/studyTagsTooltip/StudyTagsTooltip.tsx

```diff
@@ -126,11 +166,26 @@ export default class StudyTagsTooltip extends React.Component<
                 mouseEnterDelay={this.props.mouseEnterDelay}
                 placement={this.props.placement}
                 overlay={
-                    <StudyInfoOverlay
-                        studyDescription={this.props.studyDescription}
-                        studyId={this.props.studyId}
-                        isVirtualStudy={this.props.isVirtualStudy}
-                    />
+                    // for the lock icon (infoIcon false), if there are no placeholders show directly the global message
+                    // else replace the placeholders with the study tags information
+                    this.props.infoIcon === false &&
+                    getServerConfig().skin_home_page_unauthorized_studies_global_message.match(
```

Comment:
Wrap in a a message call with name f.i. hasPlaceholders(getServerConfig().skin_home_page_unauthorized_studies_global_message)